Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:12
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Copy app files to source
+COPY . .
+
+# Install initial dependencies for transpiling
+RUN npm install --save-dev @babel/core @babel/node @babel/cli @babel/preset-env
+
+# Transpile js into dist directory & copy all other files (njks templates etc)
+RUN npx babel app.js server.js -d dist && npx babel app -d dist/app --copy-files
```

Comment:
are you going to move app.js and server.js into app/ folder?

Reply:
🤦‍♂ I managed to forget about that.